VideoLocker/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -74,6 +74,13 @@ private void init() {
         // setup image cache
         createImageCache();
 
+        //setup custom font
+        CalligraphyConfig.initDefault(new CalligraphyConfig.Builder()
```

Comment:
Does this make Font Awesome the default font for all text? That doesn't seem right?